fendermint/docker/builder.local.Dockerfile

```diff
@@ -14,8 +14,11 @@ COPY . .
 # Mounting speeds up local builds, but it doesn't get cached between builds on CI.
 # OTOH it seems like one platform build can be blocked trying to acquire a lock on the build directory,
 # so for cross builds this is probably not a good idea.
+ARG RUST_VERSION=1.81.0
 RUN --mount=type=cache,target=target \
   --mount=type=cache,target=$RUSTUP_HOME,from=rust,source=$RUSTUP_HOME \
   --mount=type=cache,target=$CARGO_HOME,from=rust,source=$CARGO_HOME \
+  rustup target add wasm32-unknown-unknown &&\
+  rustup toolchain install ${RUST_VERSION}-x86_64-unknown-linux-gnu --no-self-update &&\
```

Comment:
Apparently this is needed even though that base image toolchain is 1.81.0 since we're in the `--mount=cache` layer. This will just link the toolchain, not reinstall it.